docs/conversion/sakai_21_3-21_4_mysql_conversion.sql

```diff
@@ -0,0 +1,6 @@
+-- SAK-47291
```

Comment:
Is this the correct file to put this conversion script? I think my fix should be applied to 21.x, so next version of 21 will be 21.4